### PR TITLE
Feat/Create text and sound classes

### LINKED ISSUE #28: Create Sound class
# :white_check_mark: What's needed
You must create the Sound class.
This class only need setters and getters.

## :file_folder: Files required
- `src/Games/Sound.hpp`
- `src/Games/Sound.cpp`

### LINKED ISSUE #29: Create Text class
# :white_check_mark: What's needed
You must create the Text class.
This class only need setters and getters.

## :file_folder: Files required
- `src/Games/Text.hpp`
- `src/Games/Text.cpp`

### PR DESCRIPTION
**Pull request by Thomaltarix**
I created the Text and Sound classes for the Games.

Check details here:
- #29 
- #28

### CODE REVIEW

File: src/Games/Entity.hpp [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 #pragma once
 
-#include "IEntity.hpp"
+#include "../include/IEntity.hpp"
```
Comment (mathieurobert1) [inline src/Games/Entity.hpp:10]: Can you remove the ../include/ ?

File: src/Games/Sound.hpp [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        //setters
+        void setPathSound(const std::string &pathSound) override;
+        void setVolume(float volume) override;
+        void setStatus(int status) override;
+
+        //getters
+        std::string getPathSound() override;
+        float getVolume() const override;
+        int getStatus() const override;
+    protected:
```
Comment (mathieurobert1) [inline src/Games/Sound.hpp]: you can remove the protected section

**Review by mathieurobert1 — COMMENTED** [review]
Some points to clarify

File: src/Games/Entity.hpp (hunk 1 above)
Comment (Thomaltarix) [reply src/Games/Entity.hpp:10]: We need it at the moment to compile since we do not changed the Games makefile

File: src/Games/Sound.hpp (hunk 2 above)
Comment (Thomaltarix) [reply src/Games/Sound.hpp]: Done

**Pull request merged**